terraform/outputs.tf

```diff
@@ -371,3 +371,8 @@ output "policy_definition_associated_cosmodb_id" {
 //  description = "the name of the policy exemption"
```

Comment:
If the commented lines are not required. Can we delete it?

Reply:
no they are needed when the policy exemption tf is released and we can simply uncomment it